README.md

```diff
@@ -74,75 +74,69 @@ The documentation is generated using [Jazzy](https://github.com/realm/jazzy).
 
 ### Dynamic Framework Installation [⤴](#table-of-contents)
 
-2. Add `Buy.framework` target as a depenency by navigating to:
-  - `Your Project` and (Select your target)
-  - `Build Phases`
-  - `Target Dependencies`
-  -  Add  `Buy.framework`
-3. Link `Buy.framework` by navigating to:
-  - `Your Project` and (Select your target)
-  - `Build Phases`
-  - `Link Binary With Libraries`
-  -  Add `Buy.framework`
-4. Ensure the framework is copied into the bundle by navigating to:
-  - `Your Project` and (Select your target)
-  - `Build Phases`
-  - Add `New Copy Files Phase`
-  - Select `Destination` dropdown and chose `Frameworks`
-  - Add `Buy.framework`
-5. Import into your project files using `import Buy`
- 
+2. Add `Buy.framework` target as a dependency:
+  1. Open `Your Project` and (Select your target).
+  2. Navigate to `Build Phases` > `Target Dependencies`.
+  3. Add  `Buy.framework`.
+3. Link `Buy.framework`:
+  1. Open `Your Project` and (Select your target).
+  2. Navigate to `Build Phases` > `Link Binary With Libraries`.
+  3. Add `Buy.framework`.
+4. Make sure that the framework is copied into the bundle: by navigating to:
+  1. Open `Your Project` and (Select your target).
+  2. Navigate to `Build Phases` > `New Copy Files Phase`.
+  3. Select `Destination` dropdown and chose `Frameworks`.
+  4. Add `Buy.framework`.
+5. Import into your project files using `import Buy`.
+
 See the `Storefront` sample app for an example of how to add the `Buy` target a dependency.
 
 ### Carthage [⤴](#table-of-contents)
 
-Add the following line to your Cartfile
-
-```ruby
-github "Shopify/mobile-buy-sdk-ios"
-```
-
-Then run `carthage update`
-
-Follow the [steps to link the dynamic framework](#dynamic-framework-installation-) that Carthage produced.
-
-Import the SDK module:
-
-```swift
-import Buy
-```
+1. Add the following line to your Cartfile:
+  ```ruby
+  github "Shopify/mobile-buy-sdk-ios"
+  ```
+2. Run `carthage update`.
+3. Follow the [steps to link the dynamic framework](#dynamic-framework-installation-) that Carthage produced.
+4. Import the SDK module:
+  ```swift
+  import Buy
+  ```
 
 ### CocoaPods [⤴](#table-of-contents)
 
-Add the following line to your podfile:
-
-```ruby
-pod "Mobile-Buy-SDK"
-```
+1. Add the following line to your podfile:
+  ```ruby
+  pod "Mobile-Buy-SDK"
+  ```
+2. Run `pod install`.
+3. Import the SDK module:
+  ```swift
+  import Buy
+  ```
 
-Then run `pod install`.
+## Getting started [⤴](#table-of-contents)
 
-Import the SDK module:
+The Buy SDK is built on [GraphQL](http://graphql.org/). The SDK handles all the query generation and response parsing, exposing only typed models and compile-time checked query structures. It doesn't require you to write stringed queries, or parse JSON responses.
```

Comment:
Is it OK to call this "The Buy SDK" or should it always be "The Mobile Buy SDK"?